pkg/cmd/root/sysinfo_darwin.go

```diff
@@ -3,12 +3,14 @@
 package root
 
 import (
+	"fmt"
+
```

Comment:
I dunno if it's a Go pattern, but do we need this empty line here?

Reply:
Go linter automatically puts the new line